src/components/tree/TreeLeafNode.svelte

```diff
@@ -30,6 +30,15 @@
   class="tv_node"
   class:selected
   on:click={toggleSelected}
+  role="button"
+  tabindex="0"
+  on:keydown={(e) => {
+    if (e.key !== 'Enter' && e.key !== ' ') return;
+    e.preventDefault();
+    if (e.target != null) {
+      e.target.click();
+    }
+  }}
```

Comment:
whats the purpose of this?  it looks like an addition in functionality to allow keyboard interaction with the leftmost panel (signal legend), and not a compatibility migration.